src/App.jsx

```diff
@@ -1,35 +1,22 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useEffect, useState, useContext } from "react";
```

Comment:
Do you need all of these imports in app.jsx? I don't think you need axios either.